felles/src/main/kotlin/no/nav/emottak/crypto/FileKeyStoreConfig.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.emottak.crypto
+
+import no.nav.emottak.util.getEnvVar
+import java.io.ByteArrayInputStream
+import java.io.File
+import java.io.FileInputStream
+import java.io.InputStream
+
+class FileKeyStoreConfig(
+    keyStoreFilePath: String,
+    override val keyStorePass: CharArray,
+    override val keyStoreType: String = getEnvVar("KEYSTORE_TYPE", "PKCS12")
+) : KeyStoreConfig {
+    override val keyStoreFile: InputStream = getKeyStoreFile(keyStoreFilePath)
+
+    private fun getKeyStoreFile(path: String): InputStream {
+        return try {
+            log.debug("Getting store file from $path")
```

Comment:
Noe du selv kan vurdere: 
Vi bruker en delt logger. Lurer på om det kan være lurt å lage en egen logger for denne filen slik at det blir lettere å trace i Kibana?